app/views/core_induction_programmes/modules/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Create an additional course module</h1>
+    <%= form_with model: @course_module, url: cip_create_module_path, method: :post do |f| %>
+      <%= f.govuk_error_summary %>
+      <%= f.govuk_collection_select(
+            :course_year,
+            @course_years,
+            :id,
+            :title,
+            label: { text: "Which year would you like to add a module to for?" , class: "govuk-heading-m" },
+            hint: { text: "Course Years are listed in order" }
+          ) %>
+      <%= f.govuk_collection_select(
+            :previous_module_id,
+            @course_modules,
+            :id,
+            :term_and_title,
+            label: { text: "Where would you like the module to be placed?" , class: "govuk-heading-m" },
+            hint: { text: "The module you are creating will precede the module you are selecting" }
```

Comment:
I think this line makes things harder for you.

`The module you are creating will follow the module you are selecting` makes for a much easier logic.

Reply:
Same as response above in modules controller. I did start doing it this way but switched.

Reply:
Soo, I think you are going to have a much easier time with `edit` if you let the user select the module preceding the one you are creating. 

As a general principle, because it reflects the db structure better.

As a concrete example, because when you try adding it to `edit`, it will be really hard to show a correct pre-selected value. Also, if you select previous module, it's a lot easier to find the things you need to edit - after all, you have the id of the `previous_module`, and the only other place to change is `pevious_module.next_module`.

So you could have
```
new_module = ...
previous_module = CourseModule.find(params[:previous_module_id])
next_module = previous_module.next_module

next_module.previous_module = new_module
previous_module.next_module = new_module
```
Which seems a little simpler than going to 'previous previous module'. 

If our database stored `next_module_id` on a module, then your solution would be neater, but I arbitrarily chose to store `previous_module_id` :P